pkg/controllers/etcd.go

```diff
@@ -63,7 +89,7 @@ func (s *EtcdService) Run(ctx context.Context, ready chan<- struct{}, stopped ch
 	cmd.Dir = wd
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
-	if err := cmd.Start(); err != nil {
+	if err := cmd.Run(); err != nil {
```

Comment:
What I meant with `Start` vs `Run` is use `Start()` for executing `microshift-etcd` directly - developer (non systemd) mode and use `Run()` for transient systemd unit